### PR TITLE
ENG-240: puzzle-grid-nft-image-zoom

### PR DESCRIPTION
**Pull request by stormcloud266**
makes round images more rounder

### CODE REVIEW

File: styles/globals.css [hunk 1]
```diff
@@ -60,3 +60,7 @@
 .videoAspect {
   aspect-ratio: 16 / 9;
 }
+
+.next-image-block span {
+  display: block !important;
+}
```
Comment (illepic) [inline styles/globals.css:62]: There's a reason that span isn't block. What do we need this for?

**Review by illepic — CHANGES_REQUESTED** [review]

File: styles/globals.css (hunk 1 above)
Comment (stormcloud266) [reply styles/globals.css:62]: It's oval instead of round without it

Comment (stormcloud266) [reply styles/globals.css:62]: I can also get rid of the ovalness of inline-block using absolute on my inner div if that is a better approach. no block override

Comment (illepic) [reply styles/globals.css:62]: Let me mess with some Next Image attributes here to see what we can do.

**Review by illepic — APPROVED** [review]

**Pull request merged**